handbook/engineering/hiring/software-engineer-campaigns.md

```diff
@@ -0,0 +1,75 @@
+# Software Engineer — Full Stack / Front End
```

Comment:
I think we should align the title here with the name of the file, and the title should probably indicate this role is for the campaigns team, right?